docs/src/tool/index.md

```diff
@@ -297,25 +308,26 @@ $ reach rpc-run python3 -u ./index.py
 
 # {#ref-usage-docker-reset} `reach docker-reset`
 
-You can easily kill and remove all Docker containers by executing
+This command removes all containers and volumes from Docker.
+If you have any personal containers, those containers will also be removed.
+Containers can then be restarted in your Docker Images list. It does not uninstall any Docker images.
 
 ```cmd
 $ reach docker-reset
 ```
 
 This can be a useful thing to try if your Docker containers stop responding to requests or otherwise misbehave, or if you have updated your Reach images (with `reach update`) but those changes are not taking effect.
-This command is a loose approximation of "turning Docker off and on again."
 It will affect all Docker containers on your machine, not just those created by Reach.
 
 # {#ref-usage-upgrade} `reach upgrade`
 
-You can upgrade your Reach installation by executing
+This command downloads the most recent `reach` tool.
```

Comment:
```suggestion
This command downloads the most recent `reach` script.
```

@mattaudesse can offer any additional suggestions.

Reply:
We can ignore this suggested change, as well. See https://github.com/reach-sh/reach-lang/pull/611#discussion_r813122045 for more information.